<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>780</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="MainScreen">
   <widget class="QPushButton" name="viewSched">
    <property name="geometry">
     <rect>
      <x>33</x>
      <y>685</y>
      <width>300</width>
      <height>75</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>View Schedule</string>
    </property>
   </widget>
   <widget class="QPushButton" name="manSched">
    <property name="geometry">
     <rect>
      <x>366</x>
      <y>685</y>
      <width>300</width>
      <height>75</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Manage Schedule</string>
    </property>
   </widget>
   <widget class="QPushButton" name="records">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>685</y>
      <width>300</width>
      <height>75</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Records</string>
    </property>
   </widget>
   <widget class="QPushButton" name="logout">
    <property name="geometry">
     <rect>
      <x>1032</x>
      <y>685</y>
      <width>300</width>
      <height>75</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Log Out</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1366</width>
      <height>650</height>
     </rect>
    </property>
    <widget class="QWidget" name="MainPage">
     <widget class="QPushButton" name="scope">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>230</y>
        <width>261</width>
        <height>141</height>
       </rect>
      </property>
      <property name="text">
       <string>Scope</string>
      </property>
     </widget>
     <widget class="QPushButton" name="washer">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>230</y>
        <width>261</width>
        <height>141</height>
       </rect>
      </property>
      <property name="text">
       <string>Washer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>410</y>
        <width>261</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>Scope(Open In Excel)</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>410</y>
        <width>261</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>Washer(Open In Excel)</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Scope">
     <widget class="QPushButton" name="mainBackButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="scopeTable">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>80</y>
        <width>1100</width>
        <height>550</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Washer">
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>230</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Washer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="mainBackButton_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
